script/github-actions/run-cypress-tests.js

```diff
@@ -1,17 +1,37 @@
 const { runCommandSync } = require('../utils');
 
-const tests = JSON.parse(process.env.TESTS);
+let tests = JSON.parse(process.env.TESTS);
 const step = Number(process.env.STEP);
 const numContainers = Number(process.env.NUM_CONTAINERS);
-const divider = Math.ceil(tests.length / numContainers);
 const appUrl = process.env.APP_URLS.split(',')[0];
 
+// The following logic checks if the longest-running test has been selected.
+// If it has been selected, it is run in its own container in the last parallel container.
+let divider;
+let longestTestIsPresent = false;
+const longestTest = /all-claims.cypress.spec.js/g;
```

Comment:
It's a little unfortunate to have to hard code special accommodation for an individual spec, but I suppose in our circumstances it is warranted. It will be interesting to see how much execution time this accommodation saves us in the future.

Reply:
Yeah, it's not a clean approach, but at the moment we have no better way of tracking the longest-running test. I used the same approach back when we were still using Jenkins for CI.